django/test/selenium.py

```diff
@@ -134,3 +139,17 @@ def disable_implicit_wait(self):
             yield
         finally:
             self.selenium.implicitly_wait(self.implicit_wait)
+
+    def is_page_accessible(self):
+        # id is used as an identifier here to accessibility test different states of
+        # the same URL, or we can restrict having only one is_page_accessible call on
+        # each unique URL
+        id = self.counter.get(self.selenium.current_url, 0)
+        self.counter[self.selenium.current_url] = id + 1
+
+        command = ["npx", "achecker", self.selenium.current_url + "#" + str(id)]
```

Comment:
The selenium test suite github actions are now broken unfortunately. I think you will need to have to setup nodejs npm and install this package. We would also need to update the instructions for running the selenium test suite
I think that's the main disadvantage of this approach as the python selenium test suite is now reliant on some javascript setup which people might find surprising :thinking:

Reply:
Yes, or we can make it optional by default - Otherwise, we can use an a11y checker that has a use script injection, generate results for all the test files, create baselines and then write some sort of logic to generate baselines with those previous results.